op-deployer/pkg/deployer/broadcaster/keyed.go

```diff
@@ -90,6 +90,9 @@ func NewKeyedBroadcaster(cfg KeyedBroadcasterOpts) (*KeyedBroadcaster, error) {
 }
 
 func (t *KeyedBroadcaster) Hook(bcast script.Broadcast) {
+	if bcast.From != t.mgr.From() {
+		panic("invalid from for broadcast")
```

Comment:
Can you have it output what the from address is for both t.mgr and bcast?

Reply:
Done in 4f6f4f26b276f047a310ee22d9ba321cf9916bdf.